src/main/java/org/motechproject/mots/domain/ModuleProgress.java

```diff
@@ -56,6 +57,7 @@
   @SortComparator(UnitProgressComparator.class)
   @Getter
   @Setter
+  @Valid
```

Comment:
you can remove valid, because there is no validation in UnitProgress, also those entities are used only in reports and will not be mapped to dto